### PR TITLE
Javascript javascript3 week3

### PR DESCRIPTION
**Pull request by oleksandra-lavryk**

### CODE REVIEW

File: javascript/javascript3/week3/main.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  const newCursorCircle = new Circle(
+    cursorX,
+    cursorY,
+    30,
+    0,
+    2 * Math.PI,
+    randomColor
+  );
+  newCursorCircle.draw();
+});
```
Comment (Falafelqueen) [inline javascript/javascript3/week3/main.js:63]: Amazing !!

File: javascript/javascript3/week3/promises_repo/main.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        `;
+        repoLi.appendChild(repoSubItem);
+      });
+      document.querySelector(".repo-container").appendChild(repoLi);
+    }
+  });
+}
+
+Promise.all([
+  fetch("https://api.github.com/search/repositories?q=user:rismita87"),
```
Comment (Falafelqueen) [inline javascript/javascript3/week3/promises_repo/main.js:23]: Great work ! You could make this one less repetitive and more dynamic by storing your classmates gitNames in a variable, and then only iterate over it passing their name into fetch("https://api.github.com/search/repositories?q=user:${gitName}")

**Review by Falafelqueen — APPROVED** [review]
All works just as it is supposed to, good job !

File: javascript/javascript3/week3/shopping_cart/main.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
+    return new Promise((resolve, reject) => {
```
Comment (Falafelqueen) [inline javascript/javascript3/week3/shopping_cart/main.js:7]: Nice, however you are not using the reject param in your promise, maybe you could implement also a scenario where the promise will be rejected and trow an error.

**Review by Falafelqueen — APPROVED** [review]

File: javascript/javascript3/week3/shopping_cart/main.js (hunk 3 above)
Comment (oleksandra-lavryk) [reply javascript/javascript3/week3/shopping_cart/main.js:7]: @Falafelqueen oh, I should do that. Thanks.

File: javascript/javascript3/week3/promises_repo/main.js (hunk 2 above)
Comment (oleksandra-lavryk) [reply javascript/javascript3/week3/promises_repo/main.js:23]: @Falafelqueen I will fix :)

File: javascript/javascript3/week3/main.js (hunk 1 above)
Comment (oleksandra-lavryk) [reply javascript/javascript3/week3/main.js:63]: @Falafelqueen  Thanks a lot for reviewing!!!

File: javascript/javascript3/week3/promises_repo/main.js (hunk 2 above)
Comment (oleksandra-lavryk) [reply javascript/javascript3/week3/promises_repo/main.js:23]: @Falafelqueen but I am confused how can I iterate gitNames inside  Promise.all([])? I have to put fetch as a array item inside  Promise.all([]).

**Pull request merged**